src/adapters/mc/OsConfigResource.c

```diff
@@ -518,9 +518,6 @@ void MI_CALL OsConfigResource_Invoke_GetTargetResource(
     const char* auditPassed = "Audit passed";
     const char* auditFailed = "Audit failed. See /var/log/osconfig_nrp.*";
 
-    const char* auditPassedInvalidResourceOrRuleId = "Audit passed for an invalid resource and/or rule id. See /var/log/osconfig_nrp.*";
-    const char* auditFailedInvalidResourceOrRuleId = "Audit failed for an invalid resource and/or rule id. See /var/log/osconfig_nrp.*";
```

Comment:
Please do not remove these. Instead, implement BaselineIsValidResourceIdRuleId in Baseline library.